js/packages/web/src/views/auctionCreate/index.tsx

```diff
@@ -521,13 +521,13 @@ export const AuctionCreateView = () => {
     }
 
     try {
-      const { track } = useAnalytics();
```

Comment:
This was the original culprit. Problem 1 is that it was defined inside a function (anti-hook), problem 2 was that it was already defined earlier in the scope. Almost strange that the linter / compiler didn't flag it.

Reply:
Yeah I saw that. Need to call the hooks from the main render function of the react component :+1: